nservicebus/upgrades/5to6.md

```diff
@@ -168,3 +168,12 @@ This is no longer supported. We advise to, instead of using `IBuilder` directly,
 ## AutoSubscribe
 
 The configuration option `DoNotRequireExplicitRouting()` has been obsoleted since transports with support for centralized pubsub will always auto subscribe all events without requiring explicit routing. Transports with message driven pubsub (like MSMQ, SqlServer and AzureStorageQueues) will not subscribe properly if there is no routing specified. If you used this option you can safely be removed it.
+
+## Assembly scanning
+
+Default behaviour for assembly scanning has changed and NServiceBus is no longer scanning assemblies in nested folders.
```

Comment:
@indualagarsamy I will once have the info. See here: https://github.com/Particular/docs.particular.net/pull/764#issue-102252535